### ISSUE TITLE
Screen reader button to announce available moves is visually clipped on small screens

### ISSUE THREAD

**Issue by gbarkerz** [gbarkerz/AccessibleSolitaire#69]
Currently if there is insufficient vertical space for the three buttons near the top left corner of the app (ie the Menu button and the two optional screen reader-related buttons) the screen reader button to announce available moves is clipped.

**Comment by gbarkerz** [comment]
Closing to tidy up lists, but should be considered open and is referenced by a tracking open item.
